examples/example_build_planet.py

```diff
@@ -3,35 +3,6 @@
 # GPL v2 or later.
 
 
-'''
-example_build_planet
---------------------
-
-For Earth we have well-constrained one-dimensional density models.  This allows us to
-calculate pressure as a funcion of depth.  Furthermore, petrologic data and assumptions
-regarding the convective state of the planet allow us to estimate the temperature.
-
-For planets other than Earth we have much less information, and in particular we
-know almost nothing about the pressure and temperature in the interior.  Instead, we tend
-to have measurements of things like mass, radius, and moment-of-inertia.  We would like
-to be able to make a model of the planet's interior that is consistent with those
-measurements.
-
-However, there is a difficulty with this.  In order to know the density of the planetary
-material, we need to know the pressure and temperature.  In order to know the pressure,
-we need to know the gravity profile.  And in order to the the gravity profile, we need
-to know the density.  This is a nonlinear problem which requires us to iterate to find
-a self-consistent solution.
-
-Here we show an example that does this, using the planet Mercury as motivation.
```

Comment:
I think this was a good docstring for the manual. Could you update it rather than removing it?